//Permite hacer un catch sin parametros
try {
    let num = 23 * num2;
} catch {
    console.error("Error");
}